Podfile.lock

```diff
@@ -61,7 +61,7 @@ PODS:
     - AppAuth/Core (~> 1.4)
     - GTMSessionFetcher/Core (~> 1.5)
   - GTMSessionFetcher/Core (1.7.0)
-  - Gutenberg (1.72.0):
+  - Gutenberg (1.71.3):
```

Comment:
The Gutenberg version is being downgraded due to the fact that tag `v1.73.0-alpha1` was created while version `1.72.0` was being released (i.e. before pushing tag `v1.72.0`). This makes me think that we should create a new tag `v1.73.0-alpha2` to include both the change from `v1.73.0-alpha1` and `v1.72.0`.

Reply:
Thanks @fluiddot , I pushed up the new alpha release here https://github.com/wordpress-mobile/gutenberg-mobile/pull/4682